Tests/RemoteMessagingTests/Mappers/DefaultRemoteMessagingSurveyURLBuilderTests.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  DefaultRemoteMessagingSurveyURLBuilderTests.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+import BrowserServicesKitTestsUtils
+import RemoteMessagingTestsUtils
+@testable import Subscription
+@testable import RemoteMessaging
+
+class DefaultRemoteMessagingSurveyURLBuilderTests: XCTestCase {
```

Comment:
This test file is incomplete - it doesn't test all parameters, and the function for getting a builder object doesn't allow you to customization subscription values. I'll improve it on Friday, but otherwise the actual functionality is working as expected.